app/src/main/java/edu/uw/myuw_android/LoginActivity.kt

```diff
@@ -33,7 +23,6 @@ class LoginActivity: AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_login)
-        supportActionBar?.hide()
 
         findViewById<Button>(R.id.button).setOnClickListener {
```

Comment:
button.setOnClickListener
Also I think it's better to rename the id to something like loginButton to make it more distinct